acceptance-tests/readme.md

```diff
@@ -126,3 +128,49 @@ The easiest way to develop a new test is to extend [AbstractJUnitTest](https://g
 That way, you'll get a bunch of things already injected into your test, like the `JenkinsAcceptanceTestRule` which is 
 responsible for starting up a Jenkins instance for testing, among other things.  
 For more details, see [this doc](https://github.com/jenkinsci/acceptance-test-harness/blob/master/docs/JUNIT.md).
+
+### Setting up (matrix-based) security
+
+To set up fine-grained (project-level & matrix-based) security for tests, inject the security test helper into your test:
```

Comment:
Do you know if this works with Role-based as well (which I think is an offshoot of Matrix)?

Reply:
I haven't looked into role-based much, but this is specifically to enable/configure project-based matrix authz, which is even different in the UI from non-project-based matrix security (even though they're provided by the same plugin I think), so it's pretty specific.
Speaking of, I thought about making the security helper a bit more generic to be able to cover other authorization strategies (e.g. the non-project-based matrix authz), but then thought we can cross that bridge if/when we need it. 
Maybe I'll add that now instead.
CC: @goravjoshi